app/models/host.rb

```diff
@@ -1,16 +1,10 @@
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "util/xml")
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "util/win32")
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "metadata/linux")
```

Comment:
Be careful here, host loads the world and some of that "world" assumes these things are on the load path.  Can you load the Host model in rails console?